docs/source/manual/core.rst

```diff
@@ -924,6 +924,25 @@ single ``Logger``:
 
 .. _man-core-testing-applications:
 
+Common Logging Configuration
```

Comment:
How do you feel about the [logging section on the configuration page](http://www.dropwizard.io/1.0.6/docs/manual/configuration.html#logging)?